docs/aws/audit/shieldmonitoring/rules/securityhub_enabled.mdx

```diff
@@ -23,6 +23,226 @@ CBP,RBI_MD_ITF,RBI_UCB
+
+4. If AWS Shield is not enabled, it indicates that Security Hub is not enabled in Shield, which is a misconfiguration.
+</Accordion>
+
+<Accordion title='Using CLI'>
+1. Install and configure AWS CLI: Before you can start using AWS CLI, you need to install it on your local system. You can download it from the official AWS website. After installation, you need to configure it with your AWS account credentials. You can do this by running the command `aws configure` and then entering your AWS Access Key ID, Secret Access Key, Default region name, and Default output format when prompted.
+
+2. List all the AWS accounts: Use the following command to list all the AWS accounts:
+   ```
+   aws organizations list-accounts --output text --query 'Accounts[?Status==`ACTIVE`].Id'
+   ```
+   This command will return a list of all active AWS accounts.
+
+3. Check if AWS Security Hub is enabled: For each account returned in the previous step, run the following command to check if AWS Security Hub is enabled:
+   ```
+   aws securityhub get-enabled-standards --region <region-name>
+   ```
+   Replace `<region-name>` with the name of the region you want to check. This command will return a list of all the security standards that are enabled in the specified region for the current account.
+
+4. Check for AWS Shield: In the output of the previous command, look for `arn:aws:securityhub::<aws-account-id>:subscription/aws-foundational-security-best-practices/v/1.0.0/SHIELD.1/` which indicates that AWS Shield is enabled. If this ARN is not present in the output, it means that AWS Shield is not enabled.
+</Accordion>
+
+<Accordion title='Using Python'>
+To check if Security Hub is enabled in AWS Shield, you can use the AWS SDK for Python (Boto3). Here are the steps:
+
+1. **Setup AWS SDK for Python (Boto3):**
+   First, you need to install and configure Boto3. You can install it using pip:
+
+   ```
+   pip install boto3
+   ```
+
+   Then, configure your AWS credentials. You can do this by setting the following environment variables:
+
+   ```
+   AWS_ACCESS_KEY_ID='your_access_key'
+   AWS_SECRET_ACCESS_KEY='your_secret_key'
+   ```
+
+2. **Import Boto3 and Initialize AWS Security Hub Client:**
+   Now, you can import Boto3 in your Python script and initialize the AWS Security Hub client.
+
+   ```python
+   import boto3
+
+   # Create a SecurityHub client
+   client = boto3.client('securityhub')
+   ```
+
+3. **Check if Security Hub is Enabled:**
+   You can use the `describe_hub` method to check if Security Hub is enabled. If Security Hub is not enabled, this method will raise an exception.
+
+   ```python
+   try:
+       response = client.describe_hub()
+       print("Security Hub is enabled.")
+   except Exception as e:
+       print("Security Hub is not enabled:", str(e))
+   ```
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code?t=secrets" target='_blank'>Click here for more details</a></b>